v2/guides/operations/deployment/deploy-to-kubernetes.mdx

```diff
@@ -0,0 +1,254 @@
+</Note>
+
+You can adjust this file to include any configuration values you need based on the [v2 configuration documentation](/v2/configuration/overview).
+
+Once you have your `values.yaml` file (or to use default settings), install Flipt v2 with Helm:
+
+```bash
+# Install with custom values (requires --devel flag while in beta)
+helm install flipt-v2 flipt/flipt-v2 -f values.yaml --devel
+
+# Or install with default settings
+helm install flipt-v2 flipt/flipt-v2 --devel
+```
+
+<Note>
+  Note the chart name is `flipt-v2`, not `flipt`. This is the dedicated chart
+  for Flipt v2 which includes v2-specific configurations and defaults.
+
+The `--devel` flag is required while Flipt v2 is in beta. This flag will no longer
+be needed once we release a stable version.
+
+</Note>
+
+This command installs the Flipt v2 Helm chart into your Kubernetes cluster using the configuration options specified in your `values.yaml` file merged with the default values from the chart.
+
+### 5. Forward the Port to Access Flipt v2
+
+After successfully installing Flipt v2 via the Helm chart, you should see instructions on how to access Flipt in your terminal. The instructions will look something like this:
+
+```bash
+export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=flipt-v2,app.kubernetes.io/instance=flipt-v2" -o jsonpath="{.items[0].metadata.name}")
+export CONTAINER_PORT=$(kubectl get pod --namespace default $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+kubectl --namespace default port-forward $POD_NAME 8080:$CONTAINER_PORT
+```
+
+Execute the commands in your terminal to forward the port and access Flipt v2.
+
+You should now be able to access Flipt v2 at `http://localhost:8080`.
+
+### 6. Verify the Installation and Configuration
+
+To ensure that Flipt v2 has been correctly deployed to your Kubernetes cluster, you can check the running pods:
+
+```bash
+kubectl get pods
+```
+
+You should see the Flipt v2 pod in the list with a status of `Running`.
+
+```console
+NAME                         READY   STATUS    RESTARTS   AGE
+flipt-v2-6d64f856d7-4l5qn    1/1     Running   0          32m
+```
+
+To verify that your configuration changes were applied, you can `curl` Flipt's `/meta/config` endpoint:
+
+```bash
+curl --silent http://localhost:8080/meta/config | jq
+```
+
+In the output of this command, you should see the configuration values you set in your `values.yaml` file.
+
+```json
+{
+  "log": {
+    "level": "INFO",
+    "encoding": "json"
+  },
+  "server": {
+    "grpc_port": 9000,
+    "http_port": 8080
+  },
+  "cors": {
+    "enabled": true,
+    "allowed_origins": [
+      "http://localhost:3000",
+      "https://your-frontend-domain.com"
+    ]
+  },
+  "ui": {
+    "default_theme": "dark",
+    "topbar": {
+      "color": "#7C3AED"
+    }
+  },
+  "storage": {
+    "default": {
+      "backend": {
+        "type": "memory"
+      }
+    }
+  },
+  ...
+}
+```
```

Comment:
`/meta/config` was removed in v2. I’m not sure what a good replacement would be for checking the configuration. I’d suggest either removing it or replacing it with a `/meta/info` endpoint using a different action.

Reply:
good call. I just replaced it with showing the ui with the color changes applied from config